community/index.page.tsx

```diff
@@ -124,6 +125,18 @@ const events = [
     end_date: "June 1, 2024",
     start_date: "June 1, 2024",
   },
+  {
+    name: "XRP Ledger Zone ETHDenver",
+    description:
+      "XRPL Zone: your all-in-one location for creating and collaborating on XRP Ledger (XRPL) projects. Details coming soon!",
+    type: "zone",
+    link: "http://xrplzone-ethdenver.splashthat.com",
+    location: "Denver, Colorado",
+    date: "February 27, 2024",
+    image: require("../static/img/events/XRPLZone.png.png"),
```

Comment:
Build failure caused by duplicate file extension. 
```suggestion
    image: require("../static/img/events/XRPLZone.png"),
```